azure-monitor-opentelemetry/azure/monitor/opentelemetry/util/configurations.py

```diff
@@ -15,7 +61,121 @@ def _get_configurations(**kwargs) -> Dict[str, ConfigurationValue]:
     for key, val in kwargs.items():
         configurations[key] = val
 
+    _default_exclude_instrumentations(configurations)
+    _default_disable_logging(configurations)
+    _default_disable_metrics(configurations)
+    _default_disable_tracing(configurations)
+    _default_logging_level(configurations)
+    _default_logger_name(configurations)
+    _default_logging_export_interval_ms(configurations)
+    _default_metric_readers(configurations)
+    _default_views(configurations)
+    _default_sampling_ratio(configurations)
+    _default_tracing_export_interval_ms(configurations)
+    _default_instrumentation_config(configurations)
+
+    # TODO: remove when validation added to BLRP
+    if configurations[LOGGING_EXPORT_INTERVAL_MS_ARG] <= 0:
```

Comment:
I think we can exclude this check and simply make sure to add this functionality to BLRP

Reply:
Can do. But I think it's better to leave the validation here until then. Same as the use of OTEL_BLRP_SCHEDULE_DELAY

Reply:
I felt it was best to keep it in until then just like my use of OTEL_BLRP_SCHEDULE_DELAY. But if you feel strongly, I can remove it before then.